ansible-ipi-install/roles/installer/templates/metal3-config.j2

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: metal3-config
+  namespace: openshift-machine-api
+data:
+  cache_url: ''
```

Comment:
Why isnt' this specified?   Why not do cahcing?   Is this a later enhancement?

Reply:
Adding as a feature enhancement: https://github.com/openshift-kni/baremetal-deploy/issues/131